packages/svelte/tests/runtime-runes/samples/destructure-assign-async-array-default/main.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+	let name = $state("tom");
+	let email = $state("tom@mail.com");
+	let number = $state(555);
+
+	const fetch_name = () => {
+		return new Promise((resolve) => {
+			setTimeout(() => {
+				resolve("jerry")
+			}, 100)
+		})
+	}
+
+	const fetch_email = () => {
+		return new Promise((resolve) => {
+			setTimeout(() => {
+				resolve("jerry@mail.com")
+			}, 100)
+		})
+	}
+	
+	const fetch_number = () => {
+		return new Promise((resolve) => {
```

Comment:
I think doing `Promise.resolve(777)` (and similar for the other functions) should be enough here, then we can also decrease the timeout in the test file and the test runs a bit faster overall.